### DISCUSSION TITLE
Is there a way to trigger for dependencyOverrides?

### DISCUSSION THREAD

**Discussion by Rapierian** [scala-steward-org/scala-steward#3041] [Q&A]
Hello,

I've got a set of dependencyOverrides that I'm putting in my project for transitive dependencies to eliminate eviction warnings. Since I'm not importing them directly, I don't have them as libraryDependencies. But it would still be nice to have scalaSteward open up PRs to bump their versions. Is there a setting for this?

Thanks.

**Comment by cholmes-axoni** [comment]
This MR seems like it should address this request.

https://github.com/scala-steward-org/sbt-plugin/pull/61
